resdk/resources/collection.py

```diff
@@ -28,11 +28,14 @@ class BaseCollection(BaseResource):
     WRITABLE_FIELDS = ('data', 'description', 'settings', 'descriptor_schema',
                        'descriptor') + BaseResource.WRITABLE_FIELDS
 
+    #: id's of data objects in the resource
```

Comment:
Shouldn't this be "ids of data objects in the resource (if not hydrated), data objects in the resource (if hydrated)"

NOTE: Use "ids" instead of "id's"